### PR TITLE
📝 (doc) [NO-ISSUE]: Update documentation after feedbacks

### PR DESCRIPTION
**Pull request by mbertin-ledger**
### 📝 Description
 
Update documentation after reviews

### ❓ Context

Dev portal documentation 

- **Feature**:

### ✅ Checklist

Pull Requests must pass CI checks and undergo code review. Set the PR as Draft if it is not yet ready for review.

- [ ] **Covered by automatic tests** 
- [ ] **Changeset is provided** 
- [ ] **Impact of the changes:** 
  - _list of the changes_

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.

### CODE REVIEW

File: apps/docs/pages/docs/docs.mdx [hunk 1]
```diff
@@ -18,7 +18,7 @@ You can follow the migration guidelines [here](./migrations/)
 
 ## Glossary
 
-Through all the documentation we will use some acronyme that you can find the following description :
+Throughout all the documentation we will use several acronymes that you can find the following description :
```
Comment (valpinkman) [inline apps/docs/pages/docs/docs.mdx:21]: [TYPO] acronyms

File: apps/docs/pages/docs/explanations/dmk.mdx [hunk 2]
```diff
@@ -84,8 +86,10 @@ const { observable, cancel } = await dmk.executeDeviceAction({
 ## State Management
 
 For each connected device, we are managing and providing a device state.
-The states are:
+
+The different states are:
 
 - `connected`: The device is connected.
-- `locked`: The device is locked.
+- `locked`: The device is locked. User needs to unlock it.
```
Comment (valpinkman) [inline apps/docs/pages/docs/explanations/dmk.mdx:93]: It's weird to add "User needs to unlock it", either you add something like "User needs to unlock it to perform operations with the DMK" (or something like that),or we remove that sentence

File: apps/docs/pages/docs/explanations/ledgerjs.mdx [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+Device management kit aims to replace LedgerJS libraries, mainly `hw-app-XXX` and `hw-transport-XXX` libraries
 
 ## Current Problems
 
 Ledger JS libraries where initially made for **Ledger Live** applications. As Ledger Live is a pretty old project (> 7 years),
 we have inevitably a big technical debt. Moreover time make that some part of the logic are today hard to understand and to maintain.
 
-Moreover, some device behavior are not well handled by the libraries. For example, opening an application on the device will cause unexpected disconnection.
-Another feedback we have learnt from partners (software wallets) is that we have a lack of simplicity in the libraries, it require low level knowledge to use them (ex: APDU concept).
+Some device behavior are not well handled by the libraries, for example opening an application on the device will cause unexpected disconnection.
```
Comment (valpinkman) [inline apps/docs/pages/docs/explanations/ledgerjs.mdx:10]: "not well" => "not correctly"

**Review by valpinkman — CHANGES_REQUESTED** [review]
Few typos and some some comments but overall nice

**Review by smartine-ledger — APPROVED** [review]

**Review by valpinkman — APPROVED** [review]

**Pull request merged**